.github/workflows/functional-test.yml

```diff
@@ -38,14 +43,14 @@ jobs:
     - name: Select Xcode
       uses: maxim-lobanov/setup-xcode@v1
       with:
-        xcode-version: 15.3
+        xcode-version: ${{ env.XCODE_VERSION }}
     - run: defaults write com.apple.iphonesimulator PasteboardAutomaticSync -bool false
 
     - uses: futureware-tech/simulator-action@v3
       with:
         # https://github.com/actions/runner-images/blob/main/images/macos/macos-14-arm64-Readme.md
-        model: 'iPhone 15 Plus'
-        os_version: '17.4'
+        model: ${{ env.IPHONE_MODEL }}
```

Comment:
should these values be quoted?

Reply:
Maybe either is fine? This works right now. I'm good either way.

Reply:
fine with me if it works this way